### ISSUE TITLE
Add a template for roles/some_role/README.md

### ISSUE THREAD

**Issue by alexxa** [kubevirt/kubevirt-ansible#142]
Recently there were several new PR adding new roles and every README there has a different structure and appearance. I think, it will be nice to have a template for it. 

The template should be mentioned in Contributing guidelines. 

Possible structure: 

```
## Role's name
Overview or role description

### Variables
Table/List (smth one) with variables and their short description
```

I'm not sure whether defaults should be written in that table or not. Maybe it will be easier to read and maintain docs, if they simply contain a phrase "For defaults values see defaults/main.yml"

**Comment by lukas-bednar** [comment]
I think this would be great addition to contribution guide.
I am good with your proposal, @alexxa do you think that you can close this gap ?

**Comment by alexxa** [comment]
I will add it to my long ToDO list =)

**Comment by alexxa** [comment]
Fixed by #154. Closing.
